docs/flow-logic/merging.md

```diff
@@ -8,7 +8,7 @@ description: Merge data streams in you n8n workflows.
 
 Merging allows you to bring multiple data streams together.
 
-Merging data from different branches or nodes uses the [Merge](/integrations/builtin/core-nodes/n8n-nodes-base.merge/) node. To merge data from multiple node executions, use the [Code](/integrations/builtin/core-nodes/n8n-nodes-base.code/) node.
+Merging data from different branches or nodes uses the [Merge](/integrations/builtin/core-nodes/n8n-nodes-base.merge/) node. To merge data from multiple node executions in a single node or multiple nodes, use the [Code](/integrations/builtin/core-nodes/n8n-nodes-base.code/) node.
```

Comment:
Can you update this to use "streams" rather than "branches", now that we're consistently using "streams" in the Merge node itself?

Reply:
done!